tests/PHPStan/Analyser/data/native-expressions.php

```diff
@@ -3,6 +3,7 @@
 namespace NativeExpressions;
 
 use function PHPStan\Testing\assertType;
+use function PHPStan\Testing\assertNativeType;
```

Comment:
@ondrejmirtes wdyt about adding a rule/check which verifies that tests included by `TypeInferenceTestCase` contain proper full-qualified method calls or imports when `assertType` or `assertNativeType` is contained?

Reply:
@staabm We can make `TypeInferenceTestCase` crash on the supported functions with correct names but without (or with a wrong) namespace :)